azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/SnapshotImpl.java

```diff
@@ -325,4 +331,17 @@ public SnapshotImpl withSku(SnapshotSkuType sku) {
     protected Observable<SnapshotInner> getInnerAsync() {
         return this.manager().inner().snapshots().getByResourceGroupAsync(this.resourceGroupName(), this.name());
     }
+
+    private String buildStorageAccountId(String vhdUrl) {
```

Comment:
Naming consistent: build vs. construct. Make thing consistent is important coding convention.